Demo/UIKit/ComponentsViewController.swift

```diff
@@ -42,7 +42,7 @@ internal final class ComponentsViewController: UIViewController {
     }()
 
     private lazy var applePayComponentExample: ApplePayComponentExample = {
-        let applePayComponent = ApplePayComponentExample()
+        let applePayComponentExample = ApplePayComponentExample()
```

Comment:
Uhhh good one!

Maybe better rename to something different than the name of the computed property as it could easily cause issues again in the future.

Reply:
😱 how did this happen? I think it was me who did this 😨 😱

Reply:
We have missed it on PR
Honest mistake, I happens 😅

Reply:
@goergisn Fixed [d4b05f0](https://github.com/Adyen/adyen-ios/pull/1281/commits/d4b05f0950607bdb5f7ba29dbf31252a7e933e4b)